yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm

```diff
@@ -0,0 +1,347 @@
+/mob/living/simple_animal/hostile/retaliate/goat/clown
+	name = "Gary the Goat"
+	desc = "Rather kick your butt then tell jokes."
+	icon = 'yogstation/icons/mob/clownpets.dmi'
+	icon_state = "clown_goat"
+	icon_living = "clown_goat"
+	icon_dead = "clown_goat_dead"
+	gold_core_spawnable = NO_SPAWN
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 3, /obj/item/clothing/mask/gas/clown_hat = 1, /obj/item/clothing/head/yogs/goatpelt = 1)
+
+/mob/living/simple_animal/hostile/retaliate/goat/ras
+	name = "Ralsei Goat"
+	desc = "It just wants to give you a hug!"
+	icon = 'yogstation/icons/mob/goats/ras_goat.dmi'
+	icon_state = "rasgoat"
+	icon_living = "rasgoat"
+	icon_dead = "rasgoat_dead"
+	attacktext = "''hugs''"
+	gold_core_spawnable = NO_SPAWN
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 3, /obj/item/toy/plush/goatplushie/angry = 1, /obj/item/clothing/head/yogs/goatpelt = 1)
+
+/mob/living/simple_animal/hostile/retaliate/goat/blue
+	name = "Blue Goat"
+	desc = "Im blue da da di da da bahhhh."
```

Comment:
```suggestion
	desc = "I'm blue da baah dee da baah daa."
```

The actual lyrics to the song are `I'm blue da ba dee da ba daa`, but adding the extra AH to ba makes it more goaty, I guess